### PR TITLE
Claudia 5261

### PR DESCRIPTION
**Pull request by chemaper**
#### Reviewers
 @flopezag @jframos 

 #### Description
 Claudia 5261
*Reorganization, moved tests, resources, added script to run tests
*most of the commands moved to examples
*parallelsync.py, reportstatus.py y reportsyncpending.py integrated in sync.py, now conformirng the structure there is only one command
*some minor fixes detected during testing after migration

 #### Testing
 Locally

### CODE REVIEW

**Comment by flopezag** [comment]
Changes that have to be taken:
- glancesync.conf should be moved to /conf directory
- from the following list:
  glancesync.py  
  glancesync_ami.py  
  glancesync_config.py  
  glancesync_fi.py  
  glancesync_image.py 
  glancesync_region.py  
  glancesync_serverfacade_mock.py 
  glancesync_serversfacade.py 
  sync.py
  
  Except the main one, I mean the program that call the rest of component that I assume that it is glancesync.py or sync.py, the rest should be in /glancesync directory
- You need to put a setup.py and setup.cfg files in the root directory.
- Each subdirectory should contain a **init**.py file except doc and licences directory.
- Check that both Unit tests and E2E test work after these changes.

**Comment by jframos** [comment]
Perhaps we could put the unittests in `tests/unit`. This way, we will have organised that directory like this:

```
├───tests
│   ├───acceptance
│   │   └───...
│   └───unit
│       └───...
│
```

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+python-glanceclient==0.18.0
+python-keystoneclient==1.4.0
+
```
Comment (jframos) [inline requirements.txt]: Due to you are not using the latest versions of that libs, I think you should add the **oslo.**\* lib dependencies for avoiding _import_ errors when executing operations from that libs. We have had problems in FiHelath, consecuence of that: **CLAUDIA-5316**

**Comment by flopezag** [comment]
Agree with unittests in `tests/unit`

File: scripts/run_unit_tests.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cd $(dirname $0)/../tests
+export PYTHONPATH=..
+python -m unittest discover
```
Comment (jframos) [inline scripts/run_unit_tests.sh]: Should we add a custom output path for xUnit XML report according to configuration in this other PR #4 ? 

```
  sonar.python.xunit.reportPath=target/surefire-reports/TEST-nosetests.xml
```

**Comment by chemaper** [comment]
I think all the comments are addressed. Every test has been passed.

**Comment by jframos** [comment]
Jenkins has run successfully all unit and acceptance tests with this branch.
LGTM

File: setup.cfg [hunk 3]
```diff
@@ -0,0 +1,2 @@
+[metadata]
+description-file = README.rst
```
Comment (flopezag) [inline setup.cfg]: add new line

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+description='Tool to synchronise images from a master region to other regions',
+author='Fernando Lopez Aguilar',
+author_email='fernando.lopezaguilar@telefonica.com, e.fiware.tid@telefonica.com',
+license='Apache 2.0',
+url='https://github.com/telefonicaid/fiware-glancesync',
+download_url='https://github.com/telefonicaid/fiware-glancesync/tarball/v%s' % __version__,
+keywords=['fiware', 'glancesync', 'glance',  'images', 'cloud'],
+classifiers=[
+"License :: OSI Approved :: Apache Software License", ],
+)
```
Comment (flopezag) [inline setup.py]: add new line

File: setup.cfg (hunk 3 above)
Comment (chemaper) [inline setup.cfg]: added new line

**Comment by flopezag** [comment]
LGTM

**Pull request merged**